client/src/components/Banner/Banner.stories.js

```diff
@@ -0,0 +1,129 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import Banner from "./Banner";
+import InputTextField from "../InputTextField/InputTextField";
+import Button from "../Button/Button";
+import Banner__pink from "./Banner__pink.png"
+import Banner__purpleBig from "./Banner__purple--big.png";
+
+
+
+const handleClick = e => {
+  alert("Clicked");
+};
+
+storiesOf("Banner", module)
+  .add("Home Banner", () => <Banner background={Banner__purpleBig}/>)
+  .add("Home Banner with Welcome text", () =>
+    <Banner background={Banner__purpleBig} Height='380px' FontSize='80px' LineHeight='120px' display='flex' flexdirection='column'
+    justifycontent='center' alignitems='center'>
+        <h2>Welcome, Justine</h2>
+    </Banner>
+  )
+  .add("Home Banner with Welcome Back text", () =>
+    <Banner background={Banner__purpleBig} Height='380px' FontSize='80px' LineHeight='120px' TextAlign="left" display='flex' flexdirection='column'
+    justifycontent='center' alignitems='center'> 
+      <h2>Welcome Back,<br />Justine.</h2>
+    </Banner>
+  )
+
+  .add("Community Banner", () => <Banner background={Banner__purpleBig}/>)
+  .add("Community Banner with input People", () =>
+    <Banner background={Banner__purpleBig}>
+        <h3 className="Banner__header3"> Community: People </h3>
```

Comment:
We're not going to remember this class is here, so it'd be better to remove it from the CSS and apply these styles directly to this `h3` tag